src/lib.rs

```diff
@@ -1117,14 +1130,27 @@ impl<T: Counter> Histogram<T> {
 
     /// Computes the matching histogram value for the given histogram bin.
     pub fn value_for(&self, index: usize) -> u64 {
+        // TODO what to do about indexes that are beyond what can be represented by this histogram?
+        // Pretty easy to end up shifting off the high end of a u64, e.g. asking for a big index
+        // when sigfigs is high.
```

Comment:
That's not ideal from an ergonomics perspective, I agree... but I'm not sure we really have a choice if we want this to be usable in cases where panics are unacceptable, like in FFI-heavy code. I have just such a use-case in mind, and I'd be willing to do some `Result`-wrangling to avoid a panic.